### PR TITLE
Better v1.0 updates

### PR DESCRIPTION
**Pull request by staticfloat**
This improves on Jeff's PR, and passes tests locally.  Let's see what CI says.

### CODE REVIEW

**Comment by staticfloat** [comment]
We'll re-run tests after `JLD2` tag is available

**Comment by timholy** [comment]
Anyone know who has rights on the Appveyor tests?

**Comment by timholy** [comment]
Hmm, Travis failures seem related to the changes here?

File: .travis.yml [hunk 1]
```diff
@@ -9,15 +9,15 @@ notifications:
     email: false
 script:
     - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-    - julia -e 'Pkg.clone(pwd()); Pkg.build("Colors"); Pkg.test("Colors"; coverage=true)';
+    - julia -e 'using Pkg; Pkg.clone(pwd()); Pkg.build("Colors"); Pkg.test("Colors"; coverage=true)';
```
Comment (fredrikekre) [inline .travis.yml]: Is there a reason not to use the default script?

Comment (staticfloat) [reply .travis.yml]: > Is there a reason not to use the default script?

Not as far as I can tell, I was just keeping things as similar to the previous state as possible.

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name = "Colors"
+uuid = "5ae59095-9a9b-59fe-a467-6f913c188581"
+authors = ["staticfloat@gmail.com <staticfloat@gmail.com>"]
```
Comment (fredrikekre) [inline Project.toml]: Do you want/should you have your email here?

**Comment by fredrikekre** [comment]
`AssertionError: abs(dexpect - colordiff(Lab(a...), Lab(b...), metric)) < eps_cdiff` on AV. Does `eps_cdiff` need to be bigger here?

**Comment by staticfloat** [comment]
I don't know this package well enough to answer.  Someone else who does (@timholy?) would have to answer.

**Comment by timholy** [comment]
It's a 32-bit problem, probably getting overflow. On it.

**Pull request merged**